packages/default-theme/components/SwErrorsList.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div v-if="list" class="errors-list-component">
+    <SwAlert :message="alertMessage" type="danger" />
```

Comment:
please use `{{ $t(' some string to be translated') }}` or `| translate` filter wherever it's possible ;)

Reply:
Ok, I moved that string from data object to template and added $t() function + translations in locales files.